pkg/spec/spec.go

```diff
@@ -138,3 +138,16 @@ func SplitYaml(in string) []string {
 	}
 	return out
 }
+
+// Returns the name of the first duplicate Policy spec name,
+// or empty string if no duplicates
+func (g *Group) FirstDuplicatePolicySpecName() string {
+	seen := make(map[string]bool)
```

Comment:
Nit: Usually, `map[string]struct{}` is used for sets.

Reply:
great catch, updated